packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -270,37 +270,43 @@ Future<String> _buildAotSnapshot(
     printStatus('Building app.dylib...');
 
     // These names are known to from the engine.
-    const String kDartVmIsolateSnapshotBuffer = 'kDartVmIsolateSnapshotBuffer';
-    const String kDartIsolateSnapshotBuffer = 'kDartIsolateSnapshotBuffer';
+    const String kVmIsolateSnapshot = 'kVmIsolateSnapshot';
```

Comment:
Nit: it might be more readable if you define all these variables up front - then you can have just one block of:

```
if (interpreter) {
  ...
} else {
  ...
}
```

(before the app.dylib step).

Anyway - you call; I don't feel strongly either way.